FFXIVClientStructs/FFXIV/Client/Game/UI/ContentsNote.cs

```diff
@@ -1,16 +1,16 @@
 namespace FFXIVClientStructs.FFXIV.Client.Game.UI;
 
 // Client::Game::UI::ContentsNote
-[StructLayout(LayoutKind.Explicit, Size = 0xB8)]
+[StructLayout(LayoutKind.Explicit, Size = 0xC0)]
```

Comment:
Nope, the struct size was correct with 0xB8. 🙂

```suggestion
[StructLayout(LayoutKind.Explicit, Size = 0xB8)]
```